### PR TITLE
Create a vvanaparthi home town file

### LINKED ISSUE #1: Please comment here to be added as a collaborator
Say anything!

### PR DESCRIPTION
**Pull request by vvanaparthi**
Resolves #1 
## What do u want to know ?
- do u like any specific types of food ?
- any specific types of activity you enjoy?

@githubteacher @baryal @srreanu

### CODE REVIEW

**Comment by githubteacher** [comment]
> do u like any specific types of food ?

I :heart: 🇮🇹 (:spaghetti:) and :sushi:

**Comment by srreanu** [comment]
do u like any specific types of food ?

Any good tasting food for my taste buds.

any specific types of activity you enjoy?
Sleeping

File: places/vvanaparthi-denver.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
 ## Denver, co
 ### where to eat
```
Comment (vvanaparthi) [inline places/vvanaparthi-denver.md:2]: just adding comment

Comment (vvanaparthi) [inline places/vvanaparthi-denver.md:2]: dsgdsg

**Comment by bmdangol** [comment]
Looks good. 👍

**Pull request merged**